inst/scripts/clusterSystemsScript.R

```diff
@@ -0,0 +1,87 @@
+# clusterSystemsScript.R
```

Comment:
Not sure what this script does, perhaps fill in the comments to give people a better idea?

Reply:
It doesn't do anything yet... In progress.